src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -9362,6 +9362,14 @@ private static bool IsNullabilityMismatch(TypeSymbol type1, TypeSymbol type2)
             return result;
         }
 
+        public override BoundNode? VisitUnconvertedInterpolatedString(BoundUnconvertedInterpolatedString node)
+        {
+            // This is only involved with unbound lambdas or when visiting the source of a converted tuple literal
+            var result = base.VisitInterpolatedStringBase(node);
```

Comment:
>VisitInterpolatedStringBase [](start = 30, length = 27)

Why not VisitUnconvertedInterpolatedString?

Reply:
With your other suggestion, this would now throw, so I'll leave this as is.

---
In reply to: [601832232](https://github.com/dotnet/roslyn/pull/52061#discussion_r601832232) [](ancestors = 601832232)

Reply:
Updated based on our offline discussion.

---
In reply to: [601847314](https://github.com/dotnet/roslyn/pull/52061#discussion_r601847314) [](ancestors = 601847314,601832232)